### PR TITLE
Implement search and categories

### PR DESCRIPTION
**Pull request by tungblvt**
==> Title: Implement search and categories

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/13883

### CODE REVIEW

**Comment by tungblvt** [comment]
ready

File: app/views/categories/_form.html.erb [hunk 1]
```diff
@@ -10,15 +10,10 @@
     <%= f.text_field :description, class: "form-control", placeholder: t("category.enter_description") %>
   </div>
 
-  <% if @stores %>
   <div class="form-group">
     <%= f.label :store_id %>
-    <%= f.select :store_id, options_for_select(@stores.map{ |c| [c.name, c.id]}, selected: @category.store_id), {}, class: "form-control" %>
+    <%= f.select :store_id, options_for_select(stores.map{ |c| [c.name, c.id]}, selected: @category.store_id), {}, class: "form-control" %>
```
Comment (huuphuc93) [inline app/views/categories/_form.html.erb]: stores.map{ |c| [c.name, c.id] chuyển vào helper em ơi

**Comment by tungblvt** [comment]
ready

File: app/helpers/categories_helper.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module CategoriesHelper
+  def stores
+    current_user.stores
+  end
+
+  def stores_option_for_select
+    stores.map{|c| [c.name, c.id]}
```
Comment (huuphuc93) [inline app/helpers/categories_helper.rb]:
```suggestion
    stores.pluck :name, :id
```

**Comment by tungblvt** [comment]
ready

**Pull request merged**